packages/base/graphql/queries/eventsListingMinimal.graphql

```diff
@@ -26,6 +26,7 @@ query EventsListingMinimal(
         }
       }
       date: datePublished
```

Comment:
just change this aliased field to point to startDate.

Then it also doesn't need to be treated differently in the sorting code.